Findexternpro.cmake

```diff
@@ -74,7 +74,6 @@ getCompilerPrefix(COMPILER)
 getNumBits(BITS)
 set(externpro_SIG ${externpro_REV}-${COMPILER}-${BITS})
 set(PFX86 "ProgramFiles(x86)")
-message("looking: $ENV{ProgramW6432}/externpro ${externpro_SIG}")
```

Comment:
I took this out just so it matches the Findexternpro.cmake in externpro, this line was the only difference.